src/neo4j_graphrag/retrievers/hybrid.py

```diff
@@ -162,6 +165,8 @@ def get_search_results(
             top_k (int, optional): The number of neighbors to return. Defaults to 5.
             effective_search_ratio (int): Controls the candidate pool size for the vector index by multiplying top_k to balance query
                 accuracy and performance. Defaults to 1.
+            ranker (str, HybridSearchRanker): Type of ranker to order the results from retrieval.
+            alpha (Optional[float]): Weight for the vector score when using the linear ranker. Only used when ranker is 'linear'. Defaults to 0.5 if not provided.
```

Comment:
I think we should either provide a default value, or make it more explicit in the doc and error messages that `alpha` is mandatory when using the linear reranker.

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/284/commits/51de84bd0481ab79456acb1c943a0e6f8f0b03b4 Made alpha mandatory